src/py42/clients/orgs.py

```diff
@@ -1,8 +1,225 @@
+    def endpoint_monitoring_removable_media_enabled(self):
+        value = self._t_settings["org-securityTools-device-detection-enable"]["value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_removable_media_enabled.setter
+    @bool_required
+    def endpoint_monitoring_removable_media_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings["org-securityTools-device-detection-enable"] = {
+            "key": "org-securityTools-device-detection-enable",
+            "value": bool_to_str(value),
+            "locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_cloud_sync_enabled(self):
+        value = self._t_settings["org-securityTools-cloud-detection-enable"]["value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_cloud_sync_enabled.setter
+    @bool_required
+    def endpoint_monitoring_cloud_sync_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings["org-securityTools-cloud-detection-enable"] = {
+            "key": "org-securityTools-cloud-detection-enable",
+            "value": bool_to_str(value),
+            "locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_browser_and_applications_enabled(self):
+        value = self._t_settings["org-securityTools-open-file-detection-enable"][
+            "value"
+        ]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_browser_and_applications_enabled.setter
+    @bool_required
+    def endpoint_monitoring_browser_and_applications_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings["org-securityTools-open-file-detection-enable"] = {
+            "key": "org-securityTools-open-file-detection-enable",
+            "value": bool_to_str(value),
+            "locked": False,
+        }
+
+    @property
+    def endpoint_monitoring_file_metadata_enabled(self):
+        value = self._t_settings["device_fileForensics_enabled"]["value"]
+        return str_to_bool(value)
+
+    @endpoint_monitoring_file_metadata_enabled.setter
+    @bool_required
+    def endpoint_monitoring_file_metadata_enabled(self, value):
+        if value:
+            self.endpoint_monitoring_enabled = True
+        self._t_settings["device_fileForensics_enabled"] = {
+            "key": "device_fileForensics_enabled",
+            "value": bool_to_str(value),
+            "locked": False,
+        }
+
+    def update(self):
+        msg = "Updating org_id={}, org_name={}, with changes: {}".format(
+            self.org_id, self._org.maps[1]["orgName"], self.changes
+        )
+        debug.logger.debug(msg)
+        if self.changes["settings"]:
+            self._update_settings()
+        if self.changes["t_settings"]:
+            self._update_org_settings()
+
+        return "Success" if not self.errored else "Error(s) occurred."
```

Comment:
Could we return the `response` from the `put` instead of `success` ?

Reply:
Well, there could be two responses (one for `/api/Org` and one for `/api/OrgSetting`), or two exceptions, or one successful response and one exception..

Reply:
I see.

What about a named tuple containing both responses?

Reply:
I am not completely opposed to this, just generally prefer things to be functional if possible, rather than state objects. The end users could decide if they need to hold on to responses.

Reply:
If we stick with how it is now:

"Success" is good in the python interpreter but a bool would be better for better for scripts I think

Reply:
Or a generator? And then to use the Settings Manager, you would have to loop over the update call. That way works nicely with either 1 or 2 responses.

Reply:
Yeah, there's not really a "good" solution here. I do like the namedtuple idea, it could have a flag for if there was an error, and then the two response objects on it for inspection.

Reply:
I also like the idea of a namedtuple here.